spdx-utils/src/main/resources/declared-license-mapping.yml

```diff
@@ -323,6 +323,7 @@
 "Open Software License v. 3.0": OSL-3.0
 "Oracle Free Use Terms and Conditions (FUTC)": LicenseRef-ort-oracle-futc
 "Other/Proprietary License": LicenseRef-scancode-proprietary-license
+"PSF License": Python-2.0
```

Comment:
Shouldn't this be `PSF-2.0`?

Reply:
It's somewhat confusing that SPDX actually has both.

Reply:
`PSF-2.0` one of the licenses contained within `Python-2.0` and `Python-2.0` is mostly applicable to Python language packages 

Should we update declared-license-mapping.yml  to map "Python Software Foundation License": Python-2.0 and  "Python Software Foundation": Python-2.0  to `PSF-2.0`  instead of `Python-2.0`?

Reply:
I simply find it confusing if anything that contains "PSF" (or "Python Software Foundation" written out) is *not* mapped to `PSF-2.0`. What I think we should do is fix the *existing* mappings to say

    "Python Software Foundation": PSF-2.0
    "Python Software Foundation License": PSF-2.0

and also add

    "PSF License": PSF-2.0

If a package actually means `Python-2.0` they must not use the word "Foundation" as part of the declared license, and doing so would again be a case of wrong meta-data which should be fixed via a curation, IMO.

Reply:
I have added a commit to remap all "PSF" or "Python Software Foundation" to PSF-2.0.

Also added additional commits for other [Python classifiers](https://pypi.org/classifiers/) for which we did not yet have a mapping.